src/algebra/category/CommRing/basic.lean

```diff
@@ -22,7 +22,8 @@ along with the relevant forgetful functors between them.
 
 ## Implementation notes
 
-See the note [locally reducible category instances].
+See the note [locally reducible category instances] and
+the note [reducible has_coe_to_sort instances for bundled categories].
```

Comment:
```suggestion
the note [has_coe_to_sort reducible].
```

Reply:
Oops. I replaced them all consistently with the longer note name.